From b588ef0107e07b9bedd6a6333319b79543c68d98 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 3 Nov 2014 10:52:56 +0100
Subject: [PATCH 25/25] src: flag evil connections

Change the hostnames of Google services to contain some unique
identifier (xx.trk) which makes requests easier to spot in a local
Squid access.log and pcap dumps with bare DNS requests.

Also add our domain into it, to feed it through a proxy for
application-level analysis of HTTPS connections.
---
 build/mac/tweak_info_plist.py                                       | 4 ++--
 chrome/browser/chrome_process_finder_win.cc                         | 2 +-
 .../component_updater/chrome_component_updater_configurator.cc      | 2 +-
 chrome/browser/drive/drive_api_service.cc                           | 2 +-
 chrome/browser/first_run/try_chrome_dialog_view.cc                  | 2 +-
 chrome/browser/gpu/three_d_api_observer.cc                          | 2 +-
 chrome/browser/history/web_history_service.cc                       | 6 +++---
 chrome/browser/infobars/insecure_content_infobar_delegate.cc        | 2 +-
 chrome/browser/media/webrtc_log_uploader.cc                         | 2 +-
 chrome/browser/metrics/variations/variations_service.cc             | 2 +-
 chrome/browser/nacl_host/nacl_infobar_delegate.cc                   | 2 +-
 chrome/browser/net/chrome_fraudulent_certificate_reporter.cc        | 2 +-
 chrome/browser/plugins/plugin_infobar_delegates.cc                  | 6 +++---
 chrome/browser/prerender/prerender_field_trial.cc                   | 2 +-
 chrome/browser/resources/default_apps/external_extensions.json      | 4 ++--
 chrome/browser/safe_browsing/client_side_detection_service.cc       | 6 +++---
 chrome/common/extensions/extension_constants.cc                     | 2 +-
 components/crash/app/breakpad_linux.cc                              | 2 +-
 components/crash/tools/crash_service.cc                             | 2 +-
 .../domain_reliability/baked_in_configs/accounts_google_com.json    | 4 ++--
 .../domain_reliability/baked_in_configs/ad_doubleclick_net.json     | 4 ++--
 components/domain_reliability/baked_in_configs/apis_google_com.json | 4 ++--
 components/domain_reliability/baked_in_configs/c_admob_com.json     | 4 ++--
 .../domain_reliability/baked_in_configs/clients2_google_com.json    | 4 ++--
 components/domain_reliability/baked_in_configs/csi_gstatic_com.json | 4 ++--
 components/domain_reliability/baked_in_configs/ddm_google_com.json  | 4 ++--
 components/domain_reliability/baked_in_configs/docs_google_com.json | 4 ++--
 .../domain_reliability/baked_in_configs/drive_google_com.json       | 4 ++--
 components/domain_reliability/baked_in_configs/e_admob_com.json     | 4 ++--
 .../domain_reliability/baked_in_configs/fonts_googleapis_com.json   | 4 ++--
 .../baked_in_configs/googleads4_g_doubleclick_net.json              | 4 ++--
 .../baked_in_configs/googleads_g_doubleclick_net.json               | 4 ++--
 components/domain_reliability/baked_in_configs/gstatic_com.json     | 4 ++--
 components/domain_reliability/baked_in_configs/lh3_ggpht_com.json   | 4 ++--
 components/domain_reliability/baked_in_configs/lh4_ggpht_com.json   | 4 ++--
 components/domain_reliability/baked_in_configs/lh5_ggpht_com.json   | 4 ++--
 components/domain_reliability/baked_in_configs/lh6_ggpht_com.json   | 4 ++--
 components/domain_reliability/baked_in_configs/mail_google_com.json | 4 ++--
 components/domain_reliability/baked_in_configs/media_admob_com.json | 4 ++--
 .../baked_in_configs/pagead2_googlesyndication_com.json             | 4 ++--
 .../baked_in_configs/partner_googleadservices_com.json              | 4 ++--
 .../baked_in_configs/pubads_g_doubleclick_net.json                  | 4 ++--
 .../baked_in_configs/redirector_googlevideo_com.json                | 4 ++--
 components/domain_reliability/baked_in_configs/s0_2mdn_net.json     | 4 ++--
 components/domain_reliability/baked_in_configs/ssl_gstatic_com.json | 4 ++--
 components/domain_reliability/baked_in_configs/star_admob_com.json  | 4 ++--
 .../domain_reliability/baked_in_configs/star_doubleclick_net.json   | 4 ++--
 .../domain_reliability/baked_in_configs/star_g_doubleclick_net.json | 4 ++--
 components/domain_reliability/baked_in_configs/star_ggpht_com.json  | 4 ++--
 components/domain_reliability/baked_in_configs/star_google_cn.json  | 4 ++--
 .../domain_reliability/baked_in_configs/star_google_co_uk.json      | 4 ++--
 components/domain_reliability/baked_in_configs/star_google_com.json | 4 ++--
 .../domain_reliability/baked_in_configs/star_google_com_au.json     | 4 ++--
 components/domain_reliability/baked_in_configs/star_google_de.json  | 4 ++--
 components/domain_reliability/baked_in_configs/star_google_fr.json  | 4 ++--
 components/domain_reliability/baked_in_configs/star_google_it.json  | 4 ++--
 components/domain_reliability/baked_in_configs/star_google_jp.json  | 4 ++--
 components/domain_reliability/baked_in_configs/star_google_org.json | 4 ++--
 components/domain_reliability/baked_in_configs/star_google_ru.json  | 4 ++--
 .../baked_in_configs/star_googleadservices_com.json                 | 4 ++--
 .../domain_reliability/baked_in_configs/star_googleapis_com.json    | 4 ++--
 .../baked_in_configs/star_googlesyndication_com.json                | 4 ++--
 .../baked_in_configs/star_googleusercontent_com.json                | 4 ++--
 .../domain_reliability/baked_in_configs/star_googlevideo_com.json   | 4 ++--
 .../domain_reliability/baked_in_configs/star_gstatic_com.json       | 4 ++--
 components/domain_reliability/baked_in_configs/star_gvt1_com.json   | 4 ++--
 .../domain_reliability/baked_in_configs/star_youtube_com.json       | 4 ++--
 components/domain_reliability/baked_in_configs/star_ytimg_com.json  | 4 ++--
 components/domain_reliability/baked_in_configs/t0_gstatic_com.json  | 4 ++--
 components/domain_reliability/baked_in_configs/t1_gstatic_com.json  | 4 ++--
 components/domain_reliability/baked_in_configs/t2_gstatic_com.json  | 4 ++--
 components/domain_reliability/baked_in_configs/t3_gstatic_com.json  | 4 ++--
 .../baked_in_configs/themes_googleusercontent_com.json              | 4 ++--
 components/domain_reliability/baked_in_configs/www_google_com.json  | 4 ++--
 .../baked_in_configs/www_googleadservices_com.json                  | 4 ++--
 components/domain_reliability/baked_in_configs/www_gstatic_com.json | 4 ++--
 components/domain_reliability/baked_in_configs/www_youtube_com.json | 4 ++--
 components/policy/core/common/policy_loader_win.cc                  | 2 +-
 components/policy/resources/policy_templates.json                   | 6 +++---
 components/translate/core/browser/translate_url_fetcher.cc          | 1 +
 80 files changed, 148 insertions(+), 147 deletions(-)

diff --git a/build/mac/tweak_info_plist.py b/build/mac/tweak_info_plist.py
index 4a6c475..6707923 100755
--- a/build/mac/tweak_info_plist.py
+++ b/build/mac/tweak_info_plist.py
@@ -179,7 +179,7 @@ def _AddKeystoneKeys(plist, bundle_identifier):
   also requires the |bundle_identifier| argument (com.example.product)."""
   plist['KSVersion'] = plist['CFBundleShortVersionString']
   plist['KSProductID'] = bundle_identifier
-  plist['KSUpdateURL'] = 'https://tools.google.com/service/update2'
+  plist['KSUpdateURL'] = 'https://132.trk.iridiumbrowser.de/tools.google.com/service/update2'
 
   _RemoveKeys(plist, 'KSChannelID')
   for tag_suffix in _TagSuffixes():
@@ -239,7 +239,7 @@ def Main(argv):
       return 1
     _AddBreakpadKeys(plist, options.branding)
     if options.breakpad_uploads:
-      plist['BreakpadURL'] = 'https://clients2.google.com/cr/report'
+      plist['BreakpadURL'] = 'https://133.trk.iridiumbrowser.de/clients2.google.com/cr/report'
     else:
       # This allows crash dumping to a file without uploading the
       # dump, for testing purposes.  Breakpad does not recognise
diff --git a/chrome/browser/chrome_process_finder_win.cc b/chrome/browser/chrome_process_finder_win.cc
index 5824824..c5e6ec0 100644
--- a/chrome/browser/chrome_process_finder_win.cc
+++ b/chrome/browser/chrome_process_finder_win.cc
@@ -106,7 +106,7 @@ NotifyChromeResult AttemptToNotifyRunningChrome(HWND remote_window,
                                                 bool fast_start) {
   DCHECK(remote_window);
   static const char kSearchUrl[] =
-      "http://www.google.com/search?q=%s&sourceid=chrome&ie=UTF-8";
+      "http://134.trk.iridiumbrowser.de/www.google.com/search?q=%s&sourceid=chrome&ie=UTF-8";
   DWORD process_id = 0;
   DWORD thread_id = GetWindowThreadProcessId(remote_window, &process_id);
   if (!thread_id || !process_id)
diff --git a/chrome/browser/component_updater/chrome_component_updater_configurator.cc b/chrome/browser/component_updater/chrome_component_updater_configurator.cc
index f2f6dae..ef64161 100644
--- a/chrome/browser/component_updater/chrome_component_updater_configurator.cc
+++ b/chrome/browser/component_updater/chrome_component_updater_configurator.cc
@@ -48,7 +48,7 @@ extern const char kSwitchDisablePings[] = "disable-pings";
 const char kSwitchUrlSource[] = "url-source";
 
 #define COMPONENT_UPDATER_SERVICE_ENDPOINT \
-  "//clients2.google.com/service/update2"
+  "//02.trk.iridiumbrowser.de/clients2.google.com/service/update2"
 
 // The default URL for the v3 protocol service endpoint. In some cases, the
 // component updater is allowed to fall back to and alternate URL source, if
diff --git a/chrome/browser/drive/drive_api_service.cc b/chrome/browser/drive/drive_api_service.cc
index e2f42be..e0e27e1 100644
--- a/chrome/browser/drive/drive_api_service.cc
+++ b/chrome/browser/drive/drive_api_service.cc
@@ -79,7 +79,7 @@ const char kDriveScope[] = "https://www.googleapis.com/auth/drive";
 const char kDriveAppsReadonlyScope[] =
     "https://www.googleapis.com/auth/drive.apps.readonly";
 const char kDriveAppsScope[] = "https://www.googleapis.com/auth/drive.apps";
-const char kDocsListScope[] = "https://docs.google.com/feeds/";
+const char kDocsListScope[] = "https://135.trk.iridiumbrowser.de/docs.google.com/feeds/";
 
 // Mime type to create a directory.
 const char kFolderMimeType[] = "application/vnd.google-apps.folder";
diff --git a/chrome/browser/first_run/try_chrome_dialog_view.cc b/chrome/browser/first_run/try_chrome_dialog_view.cc
index d9d95b5..f00adae 100644
--- a/chrome/browser/first_run/try_chrome_dialog_view.cc
+++ b/chrome/browser/first_run/try_chrome_dialog_view.cc
@@ -36,7 +36,7 @@
 namespace {
 
 const wchar_t kHelpCenterUrl[] =
-    L"https://www.google.com/support/chrome/bin/answer.py?answer=150752";
+    L"https://136.trk.iridiumbrowser.de/www.google.com/support/chrome/bin/answer.py?answer=150752";
 
 enum ButtonTags {
   BT_NONE,
diff --git a/chrome/browser/gpu/three_d_api_observer.cc b/chrome/browser/gpu/three_d_api_observer.cc
index 0f92631..a654db4 100644
--- a/chrome/browser/gpu/three_d_api_observer.cc
+++ b/chrome/browser/gpu/three_d_api_observer.cc
@@ -147,7 +147,7 @@ base::string16 ThreeDAPIInfoBarDelegate::GetLinkText() const {
 bool ThreeDAPIInfoBarDelegate::LinkClicked(WindowOpenDisposition disposition) {
   InfoBarService::WebContentsFromInfoBar(infobar())->OpenURL(
       content::OpenURLParams(
-          GURL("https://support.google.com/chrome/?p=ib_webgl"),
+          GURL("https://137.trk.iridiumbrowser.de/support.google.com/chrome/?p=ib_webgl"),
           content::Referrer(),
           (disposition == CURRENT_TAB) ? NEW_FOREGROUND_TAB : disposition,
           ui::PAGE_TRANSITION_LINK, false));
diff --git a/chrome/browser/history/web_history_service.cc b/chrome/browser/history/web_history_service.cc
index 7f0a806..e68621d 100644
--- a/chrome/browser/history/web_history_service.cc
+++ b/chrome/browser/history/web_history_service.cc
@@ -32,13 +32,13 @@ namespace history {
 namespace {
 
 const char kHistoryOAuthScope[] =
-    "https://www.googleapis.com/auth/chromesync";
+    "https://138.trk.iridiumbrowser.de/www.googleapis.com/auth/chromesync";
 
 const char kHistoryQueryHistoryUrl[] =
-    "https://history.google.com/history/api/lookup?client=chrome";
+    "https://139.trk.iridiumbrowser.de/history.google.com/history/api/lookup?client=chrome";
 
 const char kHistoryDeleteHistoryUrl[] =
-    "https://history.google.com/history/api/delete?client=chrome";
+    "https://140.trk.iridiumbrowser.de/history.google.com/history/api/delete?client=chrome";
 
 const char kPostDataMimeType[] = "text/plain";
 
diff --git a/chrome/browser/infobars/insecure_content_infobar_delegate.cc b/chrome/browser/infobars/insecure_content_infobar_delegate.cc
index fa90ef46..7ef5c5c 100644
--- a/chrome/browser/infobars/insecure_content_infobar_delegate.cc
+++ b/chrome/browser/infobars/insecure_content_infobar_delegate.cc
@@ -112,7 +112,7 @@ bool InsecureContentInfoBarDelegate::LinkClicked(
     WindowOpenDisposition disposition) {
   InfoBarService::WebContentsFromInfoBar(infobar())->OpenURL(
       content::OpenURLParams(
-          GURL("https://www.google.com/"
+          GURL("https://141.trk.iridiumbrowser.de/www.google.com/"
                "support/chrome/bin/answer.py?answer=1342714"),
           content::Referrer(),
           (disposition == CURRENT_TAB) ? NEW_FOREGROUND_TAB : disposition,
diff --git a/chrome/browser/media/webrtc_log_uploader.cc b/chrome/browser/media/webrtc_log_uploader.cc
index 95dccc9..a132fa2 100644
--- a/chrome/browser/media/webrtc_log_uploader.cc
+++ b/chrome/browser/media/webrtc_log_uploader.cc
@@ -29,7 +29,7 @@ const int kLogCountLimit = 5;
 const uint32 kIntermediateCompressionBufferBytes = 256 * 1024;  // 256 KB
 const int kLogListLimitLines = 50;
 
-const char kUploadURL[] = "https://clients2.google.com/cr/report";
+const char kUploadURL[] = "https://01.trk.iridiumbrowser.de/clients2.google.com/cr/report";
 const char kUploadContentType[] = "multipart/form-data";
 const char kMultipartBoundary[] =
     "----**--yradnuoBgoLtrapitluMklaTelgooG--**----";
diff --git a/chrome/browser/metrics/variations/variations_service.cc b/chrome/browser/metrics/variations/variations_service.cc
index 2bc0786..e57cce7 100644
--- a/chrome/browser/metrics/variations/variations_service.cc
+++ b/chrome/browser/metrics/variations/variations_service.cc
@@ -54,7 +54,7 @@ namespace {
 
 // Default server of Variations seed info.
 const char kDefaultVariationsServerURL[] =
-    "https://clients4.google.com/chrome-variations/seed";
+    "https://142.trk.iridiumbrowser.de/clients4.google.com/chrome-variations/seed";
 const int kMaxRetrySeedFetch = 5;
 
 // TODO(mad): To be removed when we stop updating the NetworkTimeTracker.
diff --git a/chrome/browser/nacl_host/nacl_infobar_delegate.cc b/chrome/browser/nacl_host/nacl_infobar_delegate.cc
index 2836dd3..bc8eec5 100644
--- a/chrome/browser/nacl_host/nacl_infobar_delegate.cc
+++ b/chrome/browser/nacl_host/nacl_infobar_delegate.cc
@@ -53,7 +53,7 @@ base::string16 NaClInfoBarDelegate::GetLinkText() const {
 bool NaClInfoBarDelegate::LinkClicked(WindowOpenDisposition disposition) {
   InfoBarService::WebContentsFromInfoBar(infobar())->OpenURL(
       content::OpenURLParams(
-          GURL("https://support.google.com/chrome/?p=ib_nacl"),
+          GURL("https://143.trk.iridiumbrowser.de/support.google.com/chrome/?p=ib_nacl"),
           content::Referrer(),
           (disposition == CURRENT_TAB) ? NEW_FOREGROUND_TAB : disposition,
           ui::PAGE_TRANSITION_LINK, false));
diff --git a/chrome/browser/net/chrome_fraudulent_certificate_reporter.cc b/chrome/browser/net/chrome_fraudulent_certificate_reporter.cc
index 5aef283..fca449c 100644
--- a/chrome/browser/net/chrome_fraudulent_certificate_reporter.cc
+++ b/chrome/browser/net/chrome_fraudulent_certificate_reporter.cc
@@ -24,7 +24,7 @@ namespace chrome_browser_net {
 // TODO(palmer): Switch to HTTPS when the error handling delegate is more
 // sophisticated. Ultimately we plan to attempt the report on many transports.
 static const char kFraudulentCertificateUploadEndpoint[] =
-    "http://clients3.google.com/log_cert_error";
+    "http://31.trk.iridiumbrowser.de/clients3.google.com/log_cert_error";
 
 ChromeFraudulentCertificateReporter::ChromeFraudulentCertificateReporter(
     net::URLRequestContext* request_context)
diff --git a/chrome/browser/plugins/plugin_infobar_delegates.cc b/chrome/browser/plugins/plugin_infobar_delegates.cc
index 947d077..9c6ef7d 100644
--- a/chrome/browser/plugins/plugin_infobar_delegates.cc
+++ b/chrome/browser/plugins/plugin_infobar_delegates.cc
@@ -395,7 +395,7 @@ bool PluginInstallerInfoBarDelegate::LinkClicked(
   GURL url(plugin_metadata_->help_url());
   if (url.is_empty()) {
     url = GURL(
-        "https://www.google.com/support/chrome/bin/answer.py?answer=142064");
+        "https://145.trk.iridiumbrowser.de/www.google.com/support/chrome/bin/answer.py?answer=142064");
   }
   InfoBarService::WebContentsFromInfoBar(infobar())->OpenURL(
       content::OpenURLParams(
@@ -517,8 +517,8 @@ bool PluginMetroModeInfoBarDelegate::LinkClicked(
   InfoBarService::WebContentsFromInfoBar(infobar())->OpenURL(
       content::OpenURLParams(
           GURL((mode_ == MISSING_PLUGIN) ?
-              "https://support.google.com/chrome/?p=ib_display_in_desktop" :
-              "https://support.google.com/chrome/?p=ib_redirect_to_desktop"),
+              "https://146.trk.iridiumbrowser.de/support.google.com/chrome/?p=ib_display_in_desktop" :
+              "https://147.trk.iridiumbrowser.de/support.google.com/chrome/?p=ib_redirect_to_desktop"),
           content::Referrer(),
           (disposition == CURRENT_TAB) ? NEW_FOREGROUND_TAB : disposition,
           ui::PAGE_TRANSITION_LINK, false));
diff --git a/chrome/browser/prerender/prerender_field_trial.cc b/chrome/browser/prerender/prerender_field_trial.cc
index e47c767..9a3d64d 100644
--- a/chrome/browser/prerender/prerender_field_trial.cc
+++ b/chrome/browser/prerender/prerender_field_trial.cc
@@ -74,7 +74,7 @@ const char kSkipDefaultNoPrerender[] = "SkipDefaultNoPrerender";
 const char kPrerenderServiceURLPrefixParameterName[] =
     "PrerenderServiceURLPrefix";
 const char kDefaultPrerenderServiceURLPrefix[] =
-    "https://clients4.google.com/prerenderservice/?q=";
+    "https://144.trk.iridiumbrowser.de/clients4.google.com/prerenderservice/?q=";
 const int kMinPrerenderServiceTimeoutMs = 1;
 const int kMaxPrerenderServiceTimeoutMs = 10000;
 const int kDefaultPrerenderServiceTimeoutMs = 1000;
diff --git a/chrome/browser/resources/default_apps/external_extensions.json b/chrome/browser/resources/default_apps/external_extensions.json
index 52aa500..ddb9360 100644
--- a/chrome/browser/resources/default_apps/external_extensions.json
+++ b/chrome/browser/resources/default_apps/external_extensions.json
@@ -29,11 +29,11 @@
   },
   // Google Sheets
   "aapocclcgogkmnckokdopfmhonfmgoek" : {
-    "external_update_url": "https://clients2.google.com/service/update2/crx"
+    "external_update_url": "https://03.trk.iridiumbrowser.de/clients2.google.com/service/update2/crx"
   },
   // Google Slides
   "felcaaldnbdncclmgdcncolpebgiejap" : {
-    "external_update_url": "https://clients2.google.com/service/update2/crx"
+    "external_update_url": "https://04.trk.iridiumbrowser.de/clients2.google.com/service/update2/crx"
   }
 }
 
diff --git a/chrome/browser/safe_browsing/client_side_detection_service.cc b/chrome/browser/safe_browsing/client_side_detection_service.cc
index 41e4ecc..d0d2f920 100644
--- a/chrome/browser/safe_browsing/client_side_detection_service.cc
+++ b/chrome/browser/safe_browsing/client_side_detection_service.cc
@@ -76,11 +76,11 @@ const int ClientSideDetectionService::kNegativeCacheIntervalDays = 1;
 const int ClientSideDetectionService::kPositiveCacheIntervalMinutes = 30;
 
 const char ClientSideDetectionService::kClientReportPhishingUrl[] =
-    "https://sb-ssl.google.com/safebrowsing/clientreport/phishing";
+    "https://148.trk.iridiumbrowser.de/sb-ssl.google.com/safebrowsing/clientreport/phishing";
 const char ClientSideDetectionService::kClientReportMalwareUrl[] =
-    "https://sb-ssl.google.com/safebrowsing/clientreport/malware-check";
+    "https://149.trk.iridiumbrowser.de/sb-ssl.google.com/safebrowsing/clientreport/malware-check";
 const char ClientSideDetectionService::kClientModelUrl[] =
-    "https://ssl.gstatic.com/safebrowsing/csd/client_model_v5.pb";
+    "https://150.trk.iridiumbrowser.de/ssl.gstatic.com/safebrowsing/csd/client_model_v5.pb";
 
 struct ClientSideDetectionService::ClientReportInfo {
   ClientReportPhishingRequestCallback callback;
diff --git a/chrome/common/extensions/extension_constants.cc b/chrome/common/extensions/extension_constants.cc
index 9a5c3f5..5787e6b 100644
--- a/chrome/common/extensions/extension_constants.cc
+++ b/chrome/common/extensions/extension_constants.cc
@@ -9,7 +9,7 @@ namespace extension_urls {
 namespace {
 
 const char kGalleryUpdateHttpsUrl[] =
-    "https://clients2.google.com/service/update2/crx";
+    "https://05.trk.iridiumbrowser.de/clients2.google.com/service/update2/crx";
 
 }  // namespace
 
diff --git a/components/crash/app/breakpad_linux.cc b/components/crash/app/breakpad_linux.cc
index 76f1d58..cc034df 100644
--- a/components/crash/app/breakpad_linux.cc
+++ b/components/crash/app/breakpad_linux.cc
@@ -78,7 +78,7 @@ namespace breakpad {
 namespace {
 
 #if !defined(OS_CHROMEOS)
-const char kUploadURL[] = "https://clients2.google.com/cr/report";
+const char kUploadURL[] = "https://06.trk.iridiumbrowser.de/clients2.google.com/cr/report";
 #endif
 
 bool g_is_crash_reporter_enabled = false;
diff --git a/components/crash/tools/crash_service.cc b/components/crash/tools/crash_service.cc
index 48dcbc8..db8102d 100644
--- a/components/crash/tools/crash_service.cc
+++ b/components/crash/tools/crash_service.cc
@@ -24,7 +24,7 @@ namespace {
 
 const wchar_t kTestPipeName[] = L"\\\\.\\pipe\\ChromeCrashServices";
 
-const wchar_t kCrashReportURL[] = L"https://clients2.google.com/cr/report";
+const wchar_t kCrashReportURL[] = L"https://07.trk.iridiumbrowser.de/clients2.google.com/cr/report";
 const wchar_t kCheckPointFile[] = L"crash_checkpoint.txt";
 
 typedef std::map<std::wstring, std::wstring> CrashMap;
diff --git a/components/domain_reliability/baked_in_configs/accounts_google_com.json b/components/domain_reliability/baked_in_configs/accounts_google_com.json
index d66613a..0c84e73 100644
--- a/components/domain_reliability/baked_in_configs/accounts_google_com.json
+++ b/components/domain_reliability/baked_in_configs/accounts_google_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "accounts.google.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://28.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://29.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/ad_doubleclick_net.json b/components/domain_reliability/baked_in_configs/ad_doubleclick_net.json
index e042c4a..e5513ae 100644
--- a/components/domain_reliability/baked_in_configs/ad_doubleclick_net.json
+++ b/components/domain_reliability/baked_in_configs/ad_doubleclick_net.json
@@ -4,10 +4,10 @@
   "monitored_domain": "ad.doubleclick.net",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://30.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://32.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/apis_google_com.json b/components/domain_reliability/baked_in_configs/apis_google_com.json
index 6f442ea..20daf2b 100644
--- a/components/domain_reliability/baked_in_configs/apis_google_com.json
+++ b/components/domain_reliability/baked_in_configs/apis_google_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "apis.google.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://08.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://21.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/c_admob_com.json b/components/domain_reliability/baked_in_configs/c_admob_com.json
index 1de2ce7..7f1b14f 100644
--- a/components/domain_reliability/baked_in_configs/c_admob_com.json
+++ b/components/domain_reliability/baked_in_configs/c_admob_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "c.admob.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://33.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://34.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/clients2_google_com.json b/components/domain_reliability/baked_in_configs/clients2_google_com.json
index 18982df2..3f875a5 100644
--- a/components/domain_reliability/baked_in_configs/clients2_google_com.json
+++ b/components/domain_reliability/baked_in_configs/clients2_google_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "clients2.google.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://35.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://36.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/csi_gstatic_com.json b/components/domain_reliability/baked_in_configs/csi_gstatic_com.json
index 2cb1e82c..4ada44d 100644
--- a/components/domain_reliability/baked_in_configs/csi_gstatic_com.json
+++ b/components/domain_reliability/baked_in_configs/csi_gstatic_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "csi.gstatic.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://37.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://38.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/ddm_google_com.json b/components/domain_reliability/baked_in_configs/ddm_google_com.json
index 4a33380..1eb882d 100644
--- a/components/domain_reliability/baked_in_configs/ddm_google_com.json
+++ b/components/domain_reliability/baked_in_configs/ddm_google_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "ddm.google.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://09.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://22.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/docs_google_com.json b/components/domain_reliability/baked_in_configs/docs_google_com.json
index 555d0ca..fe8a7fc 100644
--- a/components/domain_reliability/baked_in_configs/docs_google_com.json
+++ b/components/domain_reliability/baked_in_configs/docs_google_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "docs.google.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://39.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://40.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/drive_google_com.json b/components/domain_reliability/baked_in_configs/drive_google_com.json
index 7f9de68..2cd3bc1 100644
--- a/components/domain_reliability/baked_in_configs/drive_google_com.json
+++ b/components/domain_reliability/baked_in_configs/drive_google_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "drive.google.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://10.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://23.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/e_admob_com.json b/components/domain_reliability/baked_in_configs/e_admob_com.json
index 287ebea..61617c8 100644
--- a/components/domain_reliability/baked_in_configs/e_admob_com.json
+++ b/components/domain_reliability/baked_in_configs/e_admob_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "e.admob.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://41.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://42.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/fonts_googleapis_com.json b/components/domain_reliability/baked_in_configs/fonts_googleapis_com.json
index cb43d9a..73f936e 100644
--- a/components/domain_reliability/baked_in_configs/fonts_googleapis_com.json
+++ b/components/domain_reliability/baked_in_configs/fonts_googleapis_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "fonts.googleapis.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://43.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://44.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/googleads4_g_doubleclick_net.json b/components/domain_reliability/baked_in_configs/googleads4_g_doubleclick_net.json
index 4b1fb5d..f2b3197 100644
--- a/components/domain_reliability/baked_in_configs/googleads4_g_doubleclick_net.json
+++ b/components/domain_reliability/baked_in_configs/googleads4_g_doubleclick_net.json
@@ -4,10 +4,10 @@
   "monitored_domain": "googleads4.g.doubleclick.net",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://45.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://46.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/googleads_g_doubleclick_net.json b/components/domain_reliability/baked_in_configs/googleads_g_doubleclick_net.json
index 78cbf19..d4af996 100644
--- a/components/domain_reliability/baked_in_configs/googleads_g_doubleclick_net.json
+++ b/components/domain_reliability/baked_in_configs/googleads_g_doubleclick_net.json
@@ -4,10 +4,10 @@
   "monitored_domain": "googleads.g.doubleclick.net",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://47.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://48.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/gstatic_com.json b/components/domain_reliability/baked_in_configs/gstatic_com.json
index f2f9c82..19642ce 100644
--- a/components/domain_reliability/baked_in_configs/gstatic_com.json
+++ b/components/domain_reliability/baked_in_configs/gstatic_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "gstatic.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://49.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://50.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/lh3_ggpht_com.json b/components/domain_reliability/baked_in_configs/lh3_ggpht_com.json
index db1b206..b16d20e 100644
--- a/components/domain_reliability/baked_in_configs/lh3_ggpht_com.json
+++ b/components/domain_reliability/baked_in_configs/lh3_ggpht_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "lh3.ggpht.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://51.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://52.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/lh4_ggpht_com.json b/components/domain_reliability/baked_in_configs/lh4_ggpht_com.json
index c645f27..70ecdad 100644
--- a/components/domain_reliability/baked_in_configs/lh4_ggpht_com.json
+++ b/components/domain_reliability/baked_in_configs/lh4_ggpht_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "lh4.ggpht.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://53.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://54.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/lh5_ggpht_com.json b/components/domain_reliability/baked_in_configs/lh5_ggpht_com.json
index 85e93f9..9a7fd0e 100644
--- a/components/domain_reliability/baked_in_configs/lh5_ggpht_com.json
+++ b/components/domain_reliability/baked_in_configs/lh5_ggpht_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "lh5.ggpht.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://55.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://56.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/lh6_ggpht_com.json b/components/domain_reliability/baked_in_configs/lh6_ggpht_com.json
index fbdd314..9527c80 100644
--- a/components/domain_reliability/baked_in_configs/lh6_ggpht_com.json
+++ b/components/domain_reliability/baked_in_configs/lh6_ggpht_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "lh6.ggpht.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://57.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://58.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/mail_google_com.json b/components/domain_reliability/baked_in_configs/mail_google_com.json
index d876cc5..635de94 100644
--- a/components/domain_reliability/baked_in_configs/mail_google_com.json
+++ b/components/domain_reliability/baked_in_configs/mail_google_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "mail.google.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://11.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://24.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/media_admob_com.json b/components/domain_reliability/baked_in_configs/media_admob_com.json
index 12c2be0..06237e3 100644
--- a/components/domain_reliability/baked_in_configs/media_admob_com.json
+++ b/components/domain_reliability/baked_in_configs/media_admob_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "media.admob.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://59.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://60.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/pagead2_googlesyndication_com.json b/components/domain_reliability/baked_in_configs/pagead2_googlesyndication_com.json
index 747cc64..0685f38 100644
--- a/components/domain_reliability/baked_in_configs/pagead2_googlesyndication_com.json
+++ b/components/domain_reliability/baked_in_configs/pagead2_googlesyndication_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "pagead2.googlesyndication.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://61.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://62.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/partner_googleadservices_com.json b/components/domain_reliability/baked_in_configs/partner_googleadservices_com.json
index a6a918f..d6d6913 100644
--- a/components/domain_reliability/baked_in_configs/partner_googleadservices_com.json
+++ b/components/domain_reliability/baked_in_configs/partner_googleadservices_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "partner.googleadservices.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://63.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://64.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/pubads_g_doubleclick_net.json b/components/domain_reliability/baked_in_configs/pubads_g_doubleclick_net.json
index 9bcf189..41a7afe 100644
--- a/components/domain_reliability/baked_in_configs/pubads_g_doubleclick_net.json
+++ b/components/domain_reliability/baked_in_configs/pubads_g_doubleclick_net.json
@@ -4,10 +4,10 @@
   "monitored_domain": "pubads.g.doubleclick.net",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://65.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://66.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/redirector_googlevideo_com.json b/components/domain_reliability/baked_in_configs/redirector_googlevideo_com.json
index 19f6ea5..c3aff1e 100644
--- a/components/domain_reliability/baked_in_configs/redirector_googlevideo_com.json
+++ b/components/domain_reliability/baked_in_configs/redirector_googlevideo_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "redirector.googlevideo.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://67.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://68.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/s0_2mdn_net.json b/components/domain_reliability/baked_in_configs/s0_2mdn_net.json
index 427bdd2..6f02f0f 100644
--- a/components/domain_reliability/baked_in_configs/s0_2mdn_net.json
+++ b/components/domain_reliability/baked_in_configs/s0_2mdn_net.json
@@ -4,10 +4,10 @@
   "monitored_domain": "s0.2mdn.net",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://70.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://71.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/ssl_gstatic_com.json b/components/domain_reliability/baked_in_configs/ssl_gstatic_com.json
index 8abf57a..561236c 100644
--- a/components/domain_reliability/baked_in_configs/ssl_gstatic_com.json
+++ b/components/domain_reliability/baked_in_configs/ssl_gstatic_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "ssl.gstatic.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://12.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://25.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_admob_com.json b/components/domain_reliability/baked_in_configs/star_admob_com.json
index b0e513d..314b5eb 100644
--- a/components/domain_reliability/baked_in_configs/star_admob_com.json
+++ b/components/domain_reliability/baked_in_configs/star_admob_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.admob.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://72.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://73.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_doubleclick_net.json b/components/domain_reliability/baked_in_configs/star_doubleclick_net.json
index f1e3617..0f75436 100644
--- a/components/domain_reliability/baked_in_configs/star_doubleclick_net.json
+++ b/components/domain_reliability/baked_in_configs/star_doubleclick_net.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.doubleclick.net",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://74.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://75.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_g_doubleclick_net.json b/components/domain_reliability/baked_in_configs/star_g_doubleclick_net.json
index 7e6cbd6..3d5348cba 100644
--- a/components/domain_reliability/baked_in_configs/star_g_doubleclick_net.json
+++ b/components/domain_reliability/baked_in_configs/star_g_doubleclick_net.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.g.doubleclick.net",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://76.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://77.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_ggpht_com.json b/components/domain_reliability/baked_in_configs/star_ggpht_com.json
index 9085235..901e5b3 100644
--- a/components/domain_reliability/baked_in_configs/star_ggpht_com.json
+++ b/components/domain_reliability/baked_in_configs/star_ggpht_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.ggpht.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://78.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://79.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_google_cn.json b/components/domain_reliability/baked_in_configs/star_google_cn.json
index 332ebb0..5810375 100644
--- a/components/domain_reliability/baked_in_configs/star_google_cn.json
+++ b/components/domain_reliability/baked_in_configs/star_google_cn.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.google.cn",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://80.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://81.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_google_co_uk.json b/components/domain_reliability/baked_in_configs/star_google_co_uk.json
index 292d1e2..ae891ed 100644
--- a/components/domain_reliability/baked_in_configs/star_google_co_uk.json
+++ b/components/domain_reliability/baked_in_configs/star_google_co_uk.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.google.co.uk",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://82.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://83.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_google_com.json b/components/domain_reliability/baked_in_configs/star_google_com.json
index f017e91..a2b2a72 100644
--- a/components/domain_reliability/baked_in_configs/star_google_com.json
+++ b/components/domain_reliability/baked_in_configs/star_google_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.google.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://84.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://85.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_google_com_au.json b/components/domain_reliability/baked_in_configs/star_google_com_au.json
index ac2d17d..514ce23 100644
--- a/components/domain_reliability/baked_in_configs/star_google_com_au.json
+++ b/components/domain_reliability/baked_in_configs/star_google_com_au.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.google.com.au",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://86.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://87.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_google_de.json b/components/domain_reliability/baked_in_configs/star_google_de.json
index 37ad7f7..ef15406 100644
--- a/components/domain_reliability/baked_in_configs/star_google_de.json
+++ b/components/domain_reliability/baked_in_configs/star_google_de.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.google.de",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://88.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://89.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_google_fr.json b/components/domain_reliability/baked_in_configs/star_google_fr.json
index 6d26930..85368d7 100644
--- a/components/domain_reliability/baked_in_configs/star_google_fr.json
+++ b/components/domain_reliability/baked_in_configs/star_google_fr.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.google.fr",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://90.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://91.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_google_it.json b/components/domain_reliability/baked_in_configs/star_google_it.json
index 3561d0a..e378420 100644
--- a/components/domain_reliability/baked_in_configs/star_google_it.json
+++ b/components/domain_reliability/baked_in_configs/star_google_it.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.google.it",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://92.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://93.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_google_jp.json b/components/domain_reliability/baked_in_configs/star_google_jp.json
index b5d34c4..b0788c7 100644
--- a/components/domain_reliability/baked_in_configs/star_google_jp.json
+++ b/components/domain_reliability/baked_in_configs/star_google_jp.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.google.jp",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://94.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://95.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_google_org.json b/components/domain_reliability/baked_in_configs/star_google_org.json
index 8ac9b30..5ff320b 100644
--- a/components/domain_reliability/baked_in_configs/star_google_org.json
+++ b/components/domain_reliability/baked_in_configs/star_google_org.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.google.org",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://96.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://97.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_google_ru.json b/components/domain_reliability/baked_in_configs/star_google_ru.json
index 566b091..af76ce8 100644
--- a/components/domain_reliability/baked_in_configs/star_google_ru.json
+++ b/components/domain_reliability/baked_in_configs/star_google_ru.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.google.ru",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://98.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://99.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_googleadservices_com.json b/components/domain_reliability/baked_in_configs/star_googleadservices_com.json
index e170a5b..62cf8c9f 100644
--- a/components/domain_reliability/baked_in_configs/star_googleadservices_com.json
+++ b/components/domain_reliability/baked_in_configs/star_googleadservices_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.googleadservices.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://100.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://101.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_googleapis_com.json b/components/domain_reliability/baked_in_configs/star_googleapis_com.json
index fee1665..c5b7c9f 100644
--- a/components/domain_reliability/baked_in_configs/star_googleapis_com.json
+++ b/components/domain_reliability/baked_in_configs/star_googleapis_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.googleapis.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://102.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://103.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_googlesyndication_com.json b/components/domain_reliability/baked_in_configs/star_googlesyndication_com.json
index e269f3e..de31f3c 100644
--- a/components/domain_reliability/baked_in_configs/star_googlesyndication_com.json
+++ b/components/domain_reliability/baked_in_configs/star_googlesyndication_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.googlesyndication.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://104.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://105.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_googleusercontent_com.json b/components/domain_reliability/baked_in_configs/star_googleusercontent_com.json
index f59251b..650b32d 100644
--- a/components/domain_reliability/baked_in_configs/star_googleusercontent_com.json
+++ b/components/domain_reliability/baked_in_configs/star_googleusercontent_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.googleusercontent.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://106.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://107.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_googlevideo_com.json b/components/domain_reliability/baked_in_configs/star_googlevideo_com.json
index d749c17..e5ab6eb 100644
--- a/components/domain_reliability/baked_in_configs/star_googlevideo_com.json
+++ b/components/domain_reliability/baked_in_configs/star_googlevideo_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.googlevideo.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://108.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://109.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_gstatic_com.json b/components/domain_reliability/baked_in_configs/star_gstatic_com.json
index f83986d..aa1064c 100644
--- a/components/domain_reliability/baked_in_configs/star_gstatic_com.json
+++ b/components/domain_reliability/baked_in_configs/star_gstatic_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.gstatic.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://110.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://111.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_gvt1_com.json b/components/domain_reliability/baked_in_configs/star_gvt1_com.json
index 0022c18..703cf53 100644
--- a/components/domain_reliability/baked_in_configs/star_gvt1_com.json
+++ b/components/domain_reliability/baked_in_configs/star_gvt1_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.gvt1.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://112.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://113.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_youtube_com.json b/components/domain_reliability/baked_in_configs/star_youtube_com.json
index b74bbd1..7eaedf5 100644
--- a/components/domain_reliability/baked_in_configs/star_youtube_com.json
+++ b/components/domain_reliability/baked_in_configs/star_youtube_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.youtube.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://114.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://115.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/star_ytimg_com.json b/components/domain_reliability/baked_in_configs/star_ytimg_com.json
index df4e2c0..60e9cb8 100644
--- a/components/domain_reliability/baked_in_configs/star_ytimg_com.json
+++ b/components/domain_reliability/baked_in_configs/star_ytimg_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "*.ytimg.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://116.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://117.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/t0_gstatic_com.json b/components/domain_reliability/baked_in_configs/t0_gstatic_com.json
index c852545..4f1111e 100644
--- a/components/domain_reliability/baked_in_configs/t0_gstatic_com.json
+++ b/components/domain_reliability/baked_in_configs/t0_gstatic_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "t0.gstatic.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://118.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://119.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/t1_gstatic_com.json b/components/domain_reliability/baked_in_configs/t1_gstatic_com.json
index 91b54f4..e7c908b 100644
--- a/components/domain_reliability/baked_in_configs/t1_gstatic_com.json
+++ b/components/domain_reliability/baked_in_configs/t1_gstatic_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "t1.gstatic.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://120.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://121.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/t2_gstatic_com.json b/components/domain_reliability/baked_in_configs/t2_gstatic_com.json
index ae5c2ad..5501277 100644
--- a/components/domain_reliability/baked_in_configs/t2_gstatic_com.json
+++ b/components/domain_reliability/baked_in_configs/t2_gstatic_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "t2.gstatic.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://122.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://123.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/t3_gstatic_com.json b/components/domain_reliability/baked_in_configs/t3_gstatic_com.json
index 5e8dd2f..3017fbd 100644
--- a/components/domain_reliability/baked_in_configs/t3_gstatic_com.json
+++ b/components/domain_reliability/baked_in_configs/t3_gstatic_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "t3.gstatic.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://124.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://125.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/themes_googleusercontent_com.json b/components/domain_reliability/baked_in_configs/themes_googleusercontent_com.json
index 6092b5f..8f8125a 100644
--- a/components/domain_reliability/baked_in_configs/themes_googleusercontent_com.json
+++ b/components/domain_reliability/baked_in_configs/themes_googleusercontent_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "themes.googleusercontent.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://126.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://127.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/www_google_com.json b/components/domain_reliability/baked_in_configs/www_google_com.json
index 793c612..a780b88 100644
--- a/components/domain_reliability/baked_in_configs/www_google_com.json
+++ b/components/domain_reliability/baked_in_configs/www_google_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "www.google.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://13.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://26.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/www_googleadservices_com.json b/components/domain_reliability/baked_in_configs/www_googleadservices_com.json
index 203ce51..dfb8356 100644
--- a/components/domain_reliability/baked_in_configs/www_googleadservices_com.json
+++ b/components/domain_reliability/baked_in_configs/www_googleadservices_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "www.googleadservices.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://128.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://129.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/www_gstatic_com.json b/components/domain_reliability/baked_in_configs/www_gstatic_com.json
index e7d3150..565e0fb 100644
--- a/components/domain_reliability/baked_in_configs/www_gstatic_com.json
+++ b/components/domain_reliability/baked_in_configs/www_gstatic_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "www.gstatic.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://130.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://131.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/domain_reliability/baked_in_configs/www_youtube_com.json b/components/domain_reliability/baked_in_configs/www_youtube_com.json
index fd70e91..cefe84b 100644
--- a/components/domain_reliability/baked_in_configs/www_youtube_com.json
+++ b/components/domain_reliability/baked_in_configs/www_youtube_com.json
@@ -4,10 +4,10 @@
   "monitored_domain": "www.youtube.com",
   "collectors": [
     {
-      "upload_url": "https://clients2.google.com/domainreliability/upload"
+      "upload_url": "https://14.trk.iridiumbrowser.de/clients2.google.com/domainreliability/upload"
     },
     {
-      "upload_url": "https://beacons.gvt2.com/domainreliability/upload"
+      "upload_url": "https://27.trk.iridiumbrowser.de/beacons.gvt2.com/domainreliability/upload"
     }
   ],
   "monitored_resources": [
diff --git a/components/policy/core/common/policy_loader_win.cc b/components/policy/core/common/policy_loader_win.cc
index d26439d..a78cdb7 100644
--- a/components/policy/core/common/policy_loader_win.cc
+++ b/components/policy/core/common/policy_loader_win.cc
@@ -69,7 +69,7 @@ const char kLegacyBrowserSupportExtensionId[] =
 
 // The web store url that is the only trusted source for extensions.
 const char kExpectedWebStoreUrl[] =
-    ";https://clients2.google.com/service/update2/crx";
+    ";https://15.trk.iridiumbrowser.de/clients2.google.com/service/update2/crx";
 // String to be prepended to each blocked entry.
 const char kBlockedExtensionPrefix[] = "[BLOCKED]";
 
diff --git a/components/policy/resources/policy_templates.json b/components/policy/resources/policy_templates.json
index acf1948..52357b2 100644
--- a/components/policy/resources/policy_templates.json
+++ b/components/policy/resources/policy_templates.json
@@ -1655,7 +1655,7 @@
             'dynamic_refresh': True,
             'per_profile': True,
           },
-          'example_value': ['lcncmkcnkcdbbanbjakcencbaoegdjlp;https://clients2.google.com/service/update2/crx'],
+          'example_value': ['lcncmkcnkcdbbanbjakcencbaoegdjlp;https://16.trk.iridiumbrowser.de/clients2.google.com/service/update2/crx'],
           'id': 34,
           'caption': '''Configure the list of force-installed extensions''',
           'desc': '''Allows you to specify a list of extensions that will be installed silently, without user interaction.
@@ -1664,7 +1664,7 @@
 
           For each item, <ph name="PRODUCT_NAME">$1<ex>Google Chrome</ex></ph> will retrieve the extension specified by the extension ID from the update service at the specified update URL and silently install it.
 
-          For example, <ph name="EXTENSION_POLICY_EXAMPLE">lcncmkcnkcdbbanbjakcencbaoegdjlp;https://clients2.google.com/service/update2/crx</ph> installs the <ph name="EXTENSION_POLICY_EXAMPLE_EXTENSION_NAME">Google SSL Web Search</ph> extension from the standard Chrome Web Store update URL. For more information about hosting extensions, see: <ph name="LINK_TO_EXTENSION_DOC2">https://developer.chrome.com/extensions/hosting</ph>.
+          For example, <ph name="EXTENSION_POLICY_EXAMPLE">lcncmkcnkcdbbanbjakcencbaoegdjlp;https://17.trk.iridiumbrowser.de/clients2.google.com/service/update2/crx</ph> installs the <ph name="EXTENSION_POLICY_EXAMPLE_EXTENSION_NAME">Google SSL Web Search</ph> extension from the standard Chrome Web Store update URL. For more information about hosting extensions, see: <ph name="LINK_TO_EXTENSION_DOC2">https://developer.chrome.com/extensions/hosting</ph>.
 
           Users will be unable to uninstall extensions that are specified by this policy. If you remove an extension from this list, then it will be automatically uninstalled by <ph name="PRODUCT_NAME">$1<ex>Google Chrome</ex></ph>. Extensions specified in this list are also automatically whitelisted for installation; the ExtensionsInstallBlacklist does not affect them.
 
@@ -4222,7 +4222,7 @@
       'features': {
         'dynamic_refresh': True,
       },
-      'example_value': [ { "extension-id": "khgabmflimjjbclkmljlpmgaleanedem", "update-url": "http://clients2.google.com/service/update2/crx" } ],
+      'example_value': [ { "extension-id": "khgabmflimjjbclkmljlpmgaleanedem", "update-url": "http://18.trk.iridiumbrowser.de/clients2.google.com/service/update2/crx" } ],
       'id': 135,
       'caption': '''List of AppPack extensions''',
       'desc': '''This policy is active in retail mode only.
diff --git a/components/translate/core/browser/translate_url_fetcher.cc b/components/translate/core/browser/translate_url_fetcher.cc
index 09704fd..d89497f 100644
--- a/components/translate/core/browser/translate_url_fetcher.cc
+++ b/components/translate/core/browser/translate_url_fetcher.cc
@@ -61,6 +61,7 @@ bool TranslateURLFetcher::Request(
   if (!extra_request_header_.empty())
     fetcher_->SetExtraRequestHeaders(extra_request_header_);
 
+  fprintf(stderr, "translator: fetching something from %s\n", url_.spec().c_str());
   fetcher_->Start();
 
   return true;
-- 
2.4.0

